src/main/resources/com/synopsys/arc/jenkins/plugins/ownership/ItemOwnershipAction/ownershipSource.properties

```diff
@@ -0,0 +1,4 @@
+warningHeader=Warning!
+warningText=The ownership is specified by another source. \
+  If you save these settings, the ownership description will be overridden.
```

Comment:
I find the "description" here quite confusing and IFAICT the term was used only internally until this point. I suggest to leave it out.

Also, am I right to assume that item can *override* ownership of containing folder but the configuration of the folder and remaining items will be untouched?